api/app/wildfire_one.py

```diff
@@ -122,6 +134,8 @@ def _is_station_valid(station) -> bool:
 
     Returns True if station is good, False is station is bad.
     """
+    # TODO: We could remove this function entirely if we queried the API using RSQL
+    # to filter out inactive stations and stations with lat/long missing
     if station['stationStatus']['id'] != 'ACTIVE':
```

Comment:
@andrea-williams can you make a ticket for this? I kinda feel like there's more functionality behind this - e.g. filtering on the map to only show crd stations, showing inactive stations etc. etc.

Reply:
@Sybrand sorry, make a ticket for what - adding additional query builders to retrieve CRD stations or inactive stations from the API? Do we have a use case for that?

Reply:
Sorry - make a ticket for filtering on the RSQL instead of here. We could do it now - but things are working, and I'd rather just get this story out of the way. (The filtering scenario may be additional context)

Reply:
done